lms/djangoapps/support/static/support/jsx/entitlements/data/api/client.js

```diff
@@ -0,0 +1,58 @@
+import 'whatwg-fetch';
+import Cookies from 'js-cookie';
+
+import { entitlementList } from './endpoints';
+
+const HEADERS = {
+  Accept: 'application/json',
+  'Content-Type': 'application/json',
+  'X-CSRFToken': Cookies.get('csrftoken'),
+}
+
+export function requestEntitlements({username}) {
```

Comment:
Taking a step back...if this is in the `entitlements/data/api/client.js` file, is the `entitlements` part of all of these method names redundant / extraneous? 

```javascript
import { requestEntitlements, createEntitlement } from './entitlements/data/api/client';

import { get, create } from './entitlements/data/api/client';

import { 
  get as requestEntitlements, 
  create as createEntitlement,
} from './entitlements/data/api/client';
```

I get (and appreciate) the added verbosity but I wonder if it's relatively unnecessary?

Reply:
We discussed in person, but I've decided to leave this as is for now, In the future additional endpoints may be added to this client, though currently it only interacts with the entitlement api